### PR TITLE
Documentation: More details

### PR DESCRIPTION
**Pull request by xoen**
- added links to Concourse pipelines used to test/deploy
- fixed formatting of environment variable doc
- fixed broken link to helm setup
- updated frontend document to mention use of MoJ Design System

### CODE REVIEW

**Comment by xoen** [comment]
Cheers Ravi ☺️

File: doc/frontend.md [hunk 1]
```diff
@@ -7,14 +7,12 @@ interface for superusers to manage other users.
 
 ## Contributing
 
-### GOV.UK Frontend
+### Design System
 
-The Frontend app makes use of the
-[GOV.UK Frontend](https://design-system.service.gov.uk/get-started/production/)
-design system to markup and style the web interface.
+The Frontend app makes use of the [GOV.UK Frontend Design System](https://design-system.service.gov.uk/get-started/production/) and the [Ministry of Justice Design System](https://github.com/ministryofjustice/moj-design-system)  to markup and style the web interface.
```
Comment (andyhd) [inline doc/frontend.md]: It's called GOV.UK Design System - that's the overall guidance, the bit we use (the npm package) is called GOV.UK Frontend.

File: doc/frontend.md [hunk 2]
```diff
@@ -7,14 +7,12 @@ interface for superusers to manage other users.
...
 
-We install GOV.UK Frontend using npm, to easily track updated releases. The npm
-package is automatically installed as part of the Docker image build.
+We install the design systems using npm, to easily track updated releases. The npm
```
Comment (andyhd) [inline doc/frontend.md]: We install the MOJ Frontend npm module, not the Design System (which is more about documentation than anything else). GOV.UK Frontend is a dependency of MOJ Frotnend, so maybe we don't need to explicitly mention it.

**Comment by xoen** [comment]
@andyhd thanks for the comments. It's a bit confusing, sorry I merged I got the blessing from Ravi and I thought it would be fine. 

Maybe you want to tweak this file? You can probably get the wording accurate as I'm not that familiar with these Design Systems as you are.

**Pull request merged**